tensorflow_federated/python/core/impl/executors/federating_executor.py

```diff
@@ -492,128 +499,29 @@ def validate_executor_placements(cls, executor_placements):
                 'Unsupported cardinality for placement "{}": {}.'.format(
                     pl, pl_cardinality))
 
-  async def _trusted_aggregator_generate_keys(self):
+  async def _move(self, arg, target_executor):
 
-    @computations.tf_computation()
-    def generate_keys():
-      pk, sk = easy_box.gen_keypair()
-      return pk.raw, sk.raw
+    enc_clients_vals = await self.channel.send(
```

Comment:
hmm, is there only one channel?

Reply:
think this is what @yanndupis was referring to re: ChannelGrid. we will need something that maps Placement tuples to Channels. I have been using this as a placeholder in my code:
```python
PlacementPair = Tuple[placement_literals.PlacementLiteral, placement_literals.PlacementLiteral]

@dataclass
class ChannelGrid:
  channel_dict: Dict[PlacementPair, Channel]

  def __getitem__(self, placements: PlacementPair):
    py_typecheck.check_type(placements, tuple)
    py_typecheck.check_len(placements, 2)
    sorted_placements = sorted(placements, key=lambda p: p.uri)
    return self.channel_dict.get(sorted_placements)
```